package.json

```diff
@@ -1,8 +1,14 @@
 {
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "**/@google/model-viewer",
+      "**/@google/model-viewer/**"
+    ]
```

Comment:
what is the reason for these nohoist?

Reply:
`@rollup/plugin-node-resolve` only looks for node_modules inside build dir, so it's necessary to put it without hoisting

Reply:
Sorry but I couldn't understand the explanation. Could you add details or reword?

Reply:
```sh
.
├── node_modules # root
│   ├── @google/model-viewer/...  # "hoist" to root
│   └── ...
├── package.json # root
├── packages
│   ├── vrm-viewer # 💥 can not resolve `@google/model-viewer` 
│   │   ├── package.json # vrm-viewer's build base dir
│   │   ├── src
│   │   └── types
└── yarn.lock
```


```sh
.
├── node_modules # root
│   └── ...
├── package.json # root 
├── packages
│   ├── vrm-viewer  # ✅ build ok
│   │   ├── node_modules # node_modules appears here by "nohoist"
│   │   │   └── @google/model-viewer/...
│   │   ├── package.json # vrm-viewer's build base dir
│   │   ├── src
│   │   └── types
└── yarn.lock
```

Reply:
I see, thank you for the explanation!

Reply:
This change no longer needs for esbuild so reverted.